python/src/trezorlib/client.py

```diff
@@ -199,22 +206,23 @@ def send_passphrase(passphrase=None, on_device=None):
                 return send_passphrase(on_device=True)
 
         # else process host-entered passphrase
+        assert isinstance(passphrase, str)
```

Comment:
please make this stronger: `if not isinstance(...): raise RuntimeError("Passphrase must be a str")`

Reply:
Done in 675b7cd99